mailpoet/mailpoet
# Replace swift mailer with PHPMailer [MAILPOET-4142]

This PR aims to remove the dependency on the Swift Mailer library that is not maintained.

I tried to reuse code from PHPMailMethod and its error mapper. Also, it was necessary to make changes to the AmazonSES method. Unfortunately, I could not find or test all possible errors for SMTP.

I will be happy with each feedback or idea!

How to test:
Configure sending with SMTP or AmazonSES and send a newsletter. Don't forget to send previews etc.

[MAILPOET-4142]

[MAILPOET-4142]: https://mailpoet.atlassian.net/browse/MAILPOET-4142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing instructions:
1. Go to MailPoet > Settings > Send with tab
2. Click to configure custom sending
3. Select SMTP (it should have already saved data for SMTP)
4. Save it
5. Send a few newsletters and test email previews
